tests/integration/test_recipes_loading.py

```diff
@@ -0,0 +1,28 @@
+"""Test recipes are well formed."""
+
+import glob
+import os
+import pytest
+
+from esmvalcore._recipe import load_raw_recipe
+
+
+def _get_recipes():
+    recipes_path = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        '..',
+        '..',
+        'esmvaltool',
+        'recipes',
+    )
+    recipes_path = os.path.abspath(recipes_path)
+    print(recipes_path)
+    recipes = glob.glob(os.path.join(recipes_path, '*.yml'))
```

Comment:
Maybe you could do something like this, so it finds all recipes also if new subdirectories are created
```python
    recipes_path = Path(esmvaltool.__file__).absolute().parent / 'recipes'
    recipes = recipes_path.glob("**/recipe*.yml")
```